backend/migrations/000018_alter_communities_table.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE communities DROP COLUMN strategies;
+ALTER TABLE communities ADD COLUMN strategies json;
```

Comment:
Couple thoughts here:
- We should use a JSONB column type so if in the future we need to write a query to search based on values in the structure. The migration should be something like `ALTER TABLE communities ALTER strategies TYPE JSONB USING data::JSONB` this will convert any existing data
- Down migration should revert the column to its original data type `VARCHAR array`